src/release_notes/formatter/record_formatter.py

```diff
@@ -0,0 +1,79 @@
+from github_integration.model.pull_request import PullRequest
+from release_notes.model.record import Record
+
+
+class RecordFormatter:
+    """
+    A class used to format records for release notes.
+    """
+
+    DEFAULT_ISSUE_PATTERN: str = "- #{number} _{title}_ implemented by {developers} in {pull_requests}\n{release_note_rows}"
```

Comment:
sure, that's one approach, but this is also nice, JFYI: https://www.geeksforgeeks.org/template-class-in-python/

Reply:
noted in Epic below related Task.